File: python/2.7/factory/pizza/ClamPizza.py
from Pizza import *

class ClamPizza(Pizza):
    def __init__(self, ingredientFactory):
        super(ClamPizza, self).__init__()
        self.ingredientFactory = ingredientFactory

    def prepare(self):
        print "Preparing %s" %(self.name)
        self.dough = self.ingredientFactory.createDough()
        self.sauce = self.ingredientFactory.createSauce()
        self.cheese = self.ingredientFactory.createCheese()
        self.clam = self.ingredientFactory.createClam()

File: php/5.3/factory/pizza/ClamPizza.php
<?
include_once("Pizza.php");

class ClamPizza extends Pizza {
    protected $ingredientFactory;

    public function __construct(PizzaIngredientFactory $ingredientFactory) {
        $this->ingredientFactory = $ingredientFactory;
    }

    function prepare() {
        echo ("Prepareing " . $this->name . "\n");
        $this->dough = $this->ingredientFactory->createDough();
        $this->sauce = $this->ingredientFactory->createSauce();
        $this->cheese = $this->ingredientFactory->createCheese();
        $this->clam = $this->ingredientFactory->createClam();
    }
}
?>